heron/tools/cli/src/python/args.py

```diff
@@ -151,6 +154,38 @@ def add_extra_launch_classpath(parser):
   '''
   parser.add_argument(
       '--extra-launch-classpath',
-      metavar='(a string; additional JVM class path for launching topology)',
-      default="")
+      metavar='CLASS_PATH',
+      default="",
+      help='Additional JVM class path for launching topology')
+  return parser
+
+def add_dry_run(parser):
+  '''
+  :param parser:
+  :return:
+  '''
+  resp_formats = ['raw', 'table']
+  available_options = ', '.join(['%s' % opt for opt in resp_formats])
+
+  def dry_run_resp_format(value):
+    if value not in resp_formats:
+      raise argparse.ArgumentTypeError(
+          'Invalid dry-run response format: %s. Available formats: %s'
+          % (value, available_options))
+    return value
+
+  parser.add_argument(
+      '--dry-run',
+      default=False,
+      action='store_true',
+      help='Enable dry-run mode')
```

Comment:
'Enable dry-run mode. Information about the command will print but no action will be taken on the topology'